### DISCUSSION TITLE
Comma Separated Artists List for use with Research?

### DISCUSSION THREAD

**Discussion by zaptrem** [beetbox/beets#4567] [Q&A] [answered]
I’m using the beets database I made to write metadata for my songs. However, there’s a problem in that it writes artists like “Kendrick Lamar Feat. U2” When I need it to do “Kendrick Lamar, U2”. Does anyone know how to change this? Alternatively, is there another field that will give me an easily programmatically interpretable list of artists? My program is categorizing Kendrick Lamar separately from Kendrick Lamar Feat. U2 which is messing a number of things up.

**Comment by JOJ0** [comment, marked as answer]
Actually I have a PR open that handles something like this but the issue being solved was the other way round. Discogs source plug does comma spearated _always_, this is considered a bug which is hereby fixed: https://github.com/beetbox/beets/pull/4515


Hope it helps anyway. The underlying mechanics available to all source plugins are capable of doing such things you requested here, once this gets merged!
HTH

**Comment by JOJ0** [comment]
I recalled another thing: Try the ftintitle plugin. It moves eg U2 to the track title. https://beets.readthedocs.io/en/latest/plugins/ftintitle.html

**Comment by wisp3rwind** [comment]
> Alternatively, is there another field that will give me an easily programmatically interpretable list of artists?

I guess the answer must be no, since we don't have https://github.com/beetbox/beets/issues/505 (yet): Any splitting of artists is necessarily heuristic and fallible.
